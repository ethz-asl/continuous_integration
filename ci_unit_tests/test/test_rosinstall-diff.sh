#!/bin/bash

echo "Testing: NONE vs abc"
../../modules/rosinstall-diff.py NONE abc.rosinstall 2>&1
echo "RETURNED $?"

echo
echo "Testing: ab vs ab"
../../modules/rosinstall-diff.py ab.rosinstall ab.rosinstall 2>&1
echo "RETURNED $?"

echo
echo "Testing: ab vs abc"
../../modules/rosinstall-diff.py ab.rosinstall abc.rosinstall  2>&1
echo "RETURNED $?"
