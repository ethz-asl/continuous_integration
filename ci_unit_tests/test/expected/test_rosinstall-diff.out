Testing: NONE vs abc
a
b
c
RETURNED 0

Testing: ab vs ab
RETURNED 0

Testing: ab vs abc
ERROR : abc.rosinstall : b(git@github.com:ethz-asl/b.git, None) differs from b(git@github.com:ethz-asl/b.git, b) (different versions / URIs for the same local-name are not supported)
c
RETURNED 255
